App/src/main/java/com/google/sps/servlets/FormHandlerServlet.java

```diff
@@ -48,9 +60,23 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
             dataIsValid = user.isValid();
         }
 
-        if (dataIsValid)
-            response.sendRedirect("/shelters.html");
-        else
+        if (dataIsValid) {
+            String parameters = "?shelters=";
+            
+            /*List<Shelter> shelters = new ArrayList<Shelter>();
+            PopulateShelters.populate(shelters);
+            */
+
+            User n = new User();
```

Comment:
Why is there a blank user? Wouldn't we need to pass the data fetched from the form to this user instance?

Reply:
No reason! I was just using this for testing purposes but I removed it and updated the method to get information from the correct user instance.